Source/gui/SNTNotificationManager.m

```diff
@@ -212,26 +269,24 @@ - (void)hashBundleBinariesForEvent:(SNTStoredEvent *)event
   // Wait a max of 5 secs for the bundle service
   // Otherwise abandon bundle hashing and display the blockable event.
   if (dispatch_semaphore_wait(sema, dispatch_time(DISPATCH_TIME_NOW, 5 * NSEC_PER_SEC))) {
-    [withController updateBlockNotification:event withBundleHash:nil];
+    completionHandler(event, nil);
     LOGE(@"Timeout connecting to bundle service");
     return;
   }
 
   [[bc remoteObjectProxy] setNotificationListener:self.notificationListener];
 
-  // NSProgress becomes current for this thread. XPC messages vend a child node to the receiver.
-  [withController.progress becomeCurrentWithPendingUnitCount:100];
+  // Progress becomes current for this thread. XPC messages vend a child node to the receiver.
+  progressHandler(100);
 
-  // Start hashing. Progress is reported to the root NSProgress
-  // (currentWindowController.progress).
+  // Start hashing. Progress is reported 
   [[bc remoteObjectProxy]
     hashBundleBinariesForEvent:event
                          reply:^(NSString *bh, NSArray<SNTStoredEvent *> *events, NSNumber *ms) {
                            // Revert to displaying the blockable event if we fail to calculate the
                            // bundle hash
                            if (!bh)
-                             return [withController updateBlockNotification:event
-                                                             withBundleHash:nil];
+                             return completionHandler(event, nil);
```

Comment:
The condition braces can be omitted only for a single line. Either of these are acceptable:

```
if (!bh) return completionHandler(event, nil);
```

```
if (!bh) {
  return completionHandler(event, nil);
}
```

(The previous code here was wrong)

Reply:
fixed with `./Testing/fix.sh`